testcases/multizone_office_complex_air/models/MultizoneOfficeComplexAir/BaseClasses/HVACSide/BaseClasses/Component.mo

```diff
@@ -2828,6 +2982,266 @@ MultizoneOfficeComplexAir.BaseClasses.HVACSide.BaseClasses.Component.AirSide.Zon
         end ZonCon;
 
       end Control;
+
+      package Examples
```

Comment:
please make example as a separate file

Reply:
Yes, I unchecked 'store as one file' for all example packages